_data/members.yaml

```diff
@@ -244,6 +244,22 @@ gnuastro:
     <a href="https://www.gnu.org/software/gnuastro/manual/html_node/Tutorials.html"
     target="_blank">tutorials</a> is the best place to start using it.
 
+ctlearn:
+   name: CTLearn
+   logo: ctlearn.png   
+   url: https://github.com/ctlearn-project
+   repositories:
+     github: https://github.com/ctlearn-project/ctlearn.git
```

Comment:
```suggestion
     github: ctlearn-project/ctlearn
```

Reply:
Suggestion committed. I also added our Slack coordinates as a first contact for students. 

Thanks!